packages/admin/admin-icons/src/CometColor.tsx

```diff
@@ -1,23 +1,31 @@
 import { SvgIcon, SvgIconProps } from "@mui/material";
 
+function getUniqueId() {
+    return Math.random().toString(36).substring(7);
+}
```

Comment:
IMO we should the uuid package for this, since we're already using it across Comet

Reply:
https://github.com/vivid-planet/comet/pull/2529/commits/0a78b501897293b367bab57444358b389c920576